app/build.gradle.kts

```diff
@@ -19,16 +19,16 @@
 @file:Suppress("UnstableApiUsage")
 
 plugins {
-    id(libs.plugins.android.application.get().pluginId)
-    id(libs.plugins.kotlin.android.get().pluginId)
```

Comment:
Cool, finally fixed it?

Reply:
Yes, removing `gradlePlugin` from `buildSrc` made able to declare it normally.